delphi-checks/src/main/java/au/com/integradev/delphi/checks/UnusedRoutineCheck.java

```diff
@@ -67,11 +74,17 @@ public DelphiCheckContext visit(RoutineNode routine, DelphiCheckContext context)
     return super.visit(routine, context);
   }
 
-  private static boolean isViolation(RoutineNode routine) {
+  private boolean isIssue(RoutineNode routine) {
```

Comment:
Not liking the PMD-hangover name, mostly
You're right that the whole codebase is doing `isViolation` though, so I'll change it back for now.

It's likely that this will all get flipped to `isIssue` in a code cleanup at some point. 🤷